py-kms/KmsDataBase.xml

```diff
@@ -17,8 +16,6 @@
     UseForEpid="true"                                       // Use this build number for EPid generation. Should be true only for RTM builds that can be Windows servers
     MayBeServer="true"                                      // Should be true for RTM builds that can be Windows servers
     UsesNDR64="true"                                        // Genuine KMS servers use NDR64 for this build
-    WinBuildIndex="0"                                       // Index of the build (for py-kms)
-    MinDate="12/10/2016"                                    // GA release date of the build (for py-kms)

```

Comment:
Are you sure these entries are not needed (have you tested it)? These could be some custom patches by the original author.

Reply:
I doubt that https://github.com/Py-KMS-Organization/py-kms/blob/35fbc766caf76697ae29d130d663825318e40fac/py-kms/pykms_PidGenerator.py#L39 will be happy about the removal. While we could make this attribute optional, I think it is there for a reason (although only for older systems). Could you please add them back where you think they should go?

I think (if MR goes through), I'll update the docs mentioning this.